sdk/eventhub/event-hubs/README.md

```diff
@@ -102,6 +102,51 @@ const consumerClient = new EventHubConsumerClient(
 
 - This constructor takes the host name and entity name of your Event Hub instance and credential that implements the TokenCredential interface. This allows you to authenticate using an Azure Active Directory principal. There are implementations of the `TokenCredential` interface available in the [@azure/identity](https://www.npmjs.com/package/@azure/identity) package. The host name is of the format `<yournamespace>.servicebus.windows.net`. When using Azure Active Directory, your principal must be assigned a role which allows access to Event Hubs, such as the Azure Event Hubs Data Owner role. For more information about using Azure Active Directory authorization with Event Hubs, please refer to [the associated documentation](https://docs.microsoft.com/en-us/azure/event-hubs/authorize-access-azure-active-directory).
 
+### Guidance around retries
+
+The `EventHubConsumerClient` and `EventHubProducerClient` accept `retryOptions`
```

Comment:
```suggestion
The `EventHubConsumerClient` and `EventHubProducerClient` accept `options` where you can set the `retryOptions`
```

Reply:
Might be worth mentioning that they're constructor parameters as well. Might be obvious but I had to go look it up just now since both seemed feasible to me.